CONTRIBUTING.md

```diff
@@ -24,7 +24,28 @@ Follow these steps to set up the plugin locally for development:
 
 4. Open QGIS and enable the plugin via the plugin manager. Verify that the custom `about` message is displayed.
 
-5. Follow along the README example!
+## Debugging within VSCode
+
+The plugin can be debugged within [Visual Studio Code](https://code.visualstudio.com/) with the help of the [debugvs QGIS plugin](https://plugins.qgis.org/plugins/debug_vs).
+
+> [!NOTE]
```

Comment:
The only thing I would add is installing debugpy. I had because I wasn't installing with the right QGIS pip: `/Applications/QGIS-LTR.app/Contents/MacOS/bin/pip3 install debugpy`